cli/src/cmd/test.rs

```diff
@@ -251,16 +268,26 @@ fn short_test_result(name: &str, result: &forge::TestResult) {
 fn test<A: ArtifactOutput + 'static>(
     builder: MultiContractRunnerBuilder,
     project: Project<A>,
-    evm_opts: EvmOpts,
+    mut evm_opts: EvmOpts,
     filter: Filter,
     json: bool,
     allow_failure: bool,
+    gas_reports: (bool, Vec<String>),
 ) -> eyre::Result<TestOutcome> {
     let verbosity = evm_opts.verbosity;
+    let gas_reporting = gas_reports.0;
+
+    if gas_reporting && evm_opts.verbosity < 3 {
+        // force evm to do tracing, but dont hit the verbosity print path
+        evm_opts.verbosity = 3;
+    }
```

Comment:
this seems hacky but OK for now? think our verbosity modes can get a bit confusing at times

Reply:
this is basically a hack to enforce tracing is enabled - yes